adm/__make_nginx_conf

```diff
@@ -167,10 +167,19 @@ lua_package_paths = \
 
 nginx_conf_file = os.path.join(os.environ['MODULE_HOME'], 'config',
                                'nginx.conf')
+
+# add all MODULE-named environment variables to nginx.conf
+list_module_environment = []
+module = os.environ['MODULE']
+for key in os.environ.keys():
+    if key.startswith(module + '_'):
+        list_module_environment.append(key)
+
```

Comment:
I think you can remove this block of code thanks to the suggested change at line 182